app/utils/chain.ts

```diff
@@ -176,3 +177,61 @@ export const parseAssetString = (asset: string): [string, string] => {
 export const isEVMAddress = (address: string): boolean => {
   return address.startsWith("0x") && address.length === 42;
 };
+
+export const getChainKeyFromChain = (chain: string): ChainKey => {
```

Comment:
this should not be necessary if we require `ChainType` as an arg instead of the chain str

Reply:
You are right but let's revisit on https://linear.app/project-chaos/issue/YLD-141/consolidate-all-chain-configuration because it's an impactful change